gcc/rust/backend/rust-compile-var-decl.h

```diff
@@ -30,39 +30,42 @@ class CompileVarDecl : public HIRCompileBase, public HIR::HIRPatternVisitor
   using HIR::HIRPatternVisitor::visit;
 
 public:
-  static ::Bvariable *compile (tree fndecl, tree translated_type,
```

Comment:
I dont believe there is enough context in your git commit message to explain whats going on here.